test/e2e/network/loadbalancer.go

```diff
@@ -889,6 +961,13 @@ var _ = common.SIGDescribe("LoadBalancers", feature.LoadBalancer, func() {
 		e2epod.SetNodeSelection(&serverPod2.Spec, nodeSelection)
 		e2epod.NewPodClient(f).CreateSync(ctx, serverPod2)
 
+		// FIXME: this test is supposed to ensure that "a LoadBalancer UDP service
```

Comment:
same as https://github.com/kubernetes/kubernetes/pull/124729/files#r1598855963

Reply:
the test uses the same source ip and source port to keep the same tuple and target always the same backend https://github.com/kubernetes/kubernetes/pull/124729/files#r1602354172